ui.frontend/src/constants.js

```diff
@@ -374,6 +374,18 @@ export const Constants = {
      * @memberof module:FormView~Constants
      * @namespace FIELD_TYPE
      */
-    FIELD_TYPE: FIELD_TYPE
+    FIELD_TYPE: FIELD_TYPE,
+
+    /**
+     * Html input type text
+     * @type {string}
+     */
+    HTML_INPUT_TYPE_TEXT : "text",
```

Comment:
Why is this constant in public namespace, this file is exposed to customers, I don't think we need such constants for something we don't own ? Please have it inline in place where it is required.